youtube_dl/__init__.py

```diff
@@ -382,8 +382,6 @@ def parse_retries(retries):
         'external_downloader_args': external_downloader_args,
         'postprocessor_args': postprocessor_args,
         'cn_verification_proxy': opts.cn_verification_proxy,
-        'geo_verification_proxy': opts.geo_verification_proxy,
```

Comment:
Odd. I did not delete that line. I never had it. But it looks like it was added 17 days ago and I only forked yesterday. I will update my branch from the latest in master, then check the diffs carefully to make sure my branch only has the change I intended.